charts/astronomer/values.yaml

```diff
@@ -22,8 +22,8 @@ images:
     tag: 3.11.5-4
     pullPolicy: IfNotPresent
   houston:
-    repository: astronomerinc/ap-houston-api
-    tag: 0.16.4
+    repository: asoldatenko/ap-houston-api
```

Comment:
Is this change in for testing only or is this intentional?  It seems like it might cause issues later if it's not all under the same org.

Reply:
correct, but we need to release new houston and replace in MY pr otherwise it fails CI job

Reply:
@andriisoldatenko should we revert this?

Reply:
@schnie done in 5e57b2ee92816b6c8029b715c1cff8edb751ccb0